client/src/components/Driver/Table.jsx

```diff
@@ -55,14 +55,26 @@ const Table = ({ routes, cancelRoute, showRoute }) => {
       <tbody {...getTableBodyProps()}>
         {rows.map((row, i) => {
           prepareRow(row);
-          return(
-            <tr {...row.getRowProps()} className="table-row">
-              <td className="cancel" onClick={cancelRoute} id={row.original.id}>X</td>
-              {row.cells.map((cell) => {
-                return <td {...cell.getCellProps()} onClick={showRoute}>{cell.render('Cell')}</td>
-              })}
-            </tr>
-          );
+
+          if (row.original.confirmed) {
+            return(
+              <tr {...row.getRowProps()} className="table-row">
+                <td className="cancel" onClick={cancelRoute} id={row.original.id}>X</td>
+                {row.cells.map((cell) => {
+                  return <td {...cell.getCellProps()} onClick={showRoute}><b>{cell.render('Cell')}</b></td>
+                })}
+              </tr>
+            );
+          } else {
+            return(
+              <tr {...row.getRowProps()} className="table-row">
+                <td className="cancel" onClick={cancelRoute} id={row.original.id}>X</td>
+                {row.cells.map((cell) => {
+                  return <td {...cell.getCellProps()} onClick={showRoute}>{cell.render('Cell')}</td>
+                })}
+              </tr>
+            );
+          }
         })}
```

Comment:
another way to do this without repeating the code would be to use a conditional `className`  dependent on the status of `row.original.confirmed`:

`<td className={row.original.confirmed ? "route-confirmed" : ""}>...</td>`

then you could just style the class with css to make it bold.

Reply:
this is a fantastic tip, thank you!